packages/paste-core/utilities/flex/src/index.tsx

```diff
@@ -0,0 +1,191 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {ResponsiveValue} from 'styled-system';
+import {Box} from '@twilio-paste/box';
+import {FlexboxProps} from '@twilio-paste/types';
+
+export type DisplayOptions = ResponsiveValue<'flex' | 'inline-flex'>;
+type VerticalAlignOptions = 'top' | 'center' | 'bottom' | 'stretch';
+export type VerticalAlign = ResponsiveValue<VerticalAlignOptions>;
+type HorizontalAlignOptions = 'left' | 'center' | 'right' | 'around' | 'between';
+export type HorizontalAlign = ResponsiveValue<HorizontalAlignOptions>;
+export type VerticalOptions = ResponsiveValue<boolean | 'column' | 'row'>;
+export type GrowOptions = ResponsiveValue<boolean | number>;
+export type ShrinkOptions = ResponsiveValue<boolean | number>;
+export type BasisOptions = ResponsiveValue<string | number>;
+export type WrapOptions = ResponsiveValue<boolean | 'wrap' | 'nowrap'>;
+
+export interface FlexProps {
+  display?: DisplayOptions;
+  vertical?: VerticalOptions;
+  vAlignContent?: VerticalAlign;
+  hAlignContent?: HorizontalAlign;
+  grow?: GrowOptions;
+  shrink?: ShrinkOptions;
+  basis?: BasisOptions;
+  wrap?: WrapOptions;
+}
+
+const getGrow = ({grow}: FlexProps): {} => {
+  if (typeof grow === 'number' || Array.isArray(grow)) {
+    return grow;
+  }
+
+  if (grow) {
+    return 1;
+  }
+
+  return 0; // default
```

Comment:
nit: I don't think these comments provide much value, they could be safely removed.